bin/common.sh

```diff
@@ -171,10 +171,6 @@ else
 fi
 export ZEPPELIN_RUNNER
 
-if [[ -z "$ZEPPELIN_IDENT_STRING" ]]; then
-  export ZEPPELIN_IDENT_STRING="${USER}"
-fi
```

Comment:
log filename seems weird after this removal when I just upgrade from zeppelin 10.0.1 to 11.0.0

```
zeppelin--SERVER.log.2024-04-24
zeppelin--SERVER.out
zeppelin-interpreter-jdbc-shared_process--SERVER.log
```